google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -57,6 +57,8 @@
                         serviceImplementation="com.google.cloud.tools.intellij.DefaultAccountPluginConfigurationService"/>
     <applicationService serviceInterface="com.google.cloud.tools.intellij.login.GoogleLoginService"
                         serviceImplementation="com.google.cloud.tools.intellij.login.IntellijGoogleLoginService"/>
+    <applicationService serviceInterface="com.google.cloud.tools.intellij.analytics.IdeaUsageTracker"
```

Comment:
You should talk to Appu about this.  The main point is that Android Studio
also wants to track our events and we don't want to duplicate tracking
calls for both properties.

On Fri, May 27, 2016 at 6:58 PM, Chanseok Oh <notifications@github.com>
wrote:

> In google-account-plugin/resources/META-INF/plugin.xml
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/699#discussion_r64935600>
> :
>
> > @@ -57,6 +57,8 @@
> >                          serviceImplementation="com.google.cloud.tools.intellij.DefaultAccountPluginConfigurationService"/>
> >      <applicationService serviceInterface="com.google.cloud.tools.intellij.login.GoogleLoginService"
> >                          serviceImplementation="com.google.cloud.tools.intellij.login.IntellijGoogleLoginService"/>
> > +    <applicationService serviceInterface="com.google.cloud.tools.intellij.analytics.IdeaUsageTracker"
>
> Hmm... I thought Android would go a separate way from here, since it will
> use its own GA property. We can't use the cloud-reporting lib for Android
> Studio. There should be different facilities for AS.
>
> What could be the best way?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/699/files/bde422900095e05a7dcf1625850ae2a11fd65cfa#r64935600>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HZdzwMMbMFQD6ncHbZmxqyucnJ2kks5qFyKsgaJpZM4ImAuk>
> .
>